src/clients/index.ts

```diff
@@ -154,6 +154,10 @@ export interface ClientProps {
    * Type of the client
    */
   type: ClientType;
+  /**
+   * Path for exchanging tokens. Leave blank to use standard path realms/<realm>/protocol/openid-connect/token
```

Comment:
Isn't it so that there's no global standard about these paths? So the mentioned "standard" is actually just Keycloak convention? So perhaps the comment could be something like "Leave blank to use the default Keycloak path realms/<realm>/protocol/openid-connect/token".